docker-selenium.yaml

```diff
@@ -173,42 +246,464 @@ subpackages:
-              cp selenium.conf ${{targets.subpkgdir}}/etc/supervisor/conf.d/
-              cp start-vnc.sh ${{targets.subpkgdir}}/opt/bin/
-              cp start-novnc.sh ${{targets.subpkgdir}}/opt/bin/
-              cp selenium_grid_logo.png ${{targets.subpkgdir}}/usr/share/images/fluxbox/ubuntu-light.png
-              cp generate_relay_config ${{targets.subpkgdir}}/opt/bin/generate_relay_config
-
-              mkdir -p ${{targets.subpkgdir}}/home/$SEL_USER/.fluxbox
-              mkdir -p ${{targets.subpkgdir}}/tmp/.X11-unix
-              mkdir -p ${{targets.subpkgdir}}/home/$SEL_USER/.vnc
-              mkdir -p ${{targets.subpkgdir}}/opt/selenium
-              mkdir -p ${{targets.subpkgdir}}/opt/selenium
-              echo "${SEL_PASSWD}" | x11vnc -storepasswd - ${{targets.subpkgdir}}/home/$SEL_USER/.vnc/passwd
-      # https://github.com/SeleniumHQ/docker-selenium/blob/trunk/NodeChrome/Dockerfile
+              mkdir -p ${{targets.contextdir}}/opt/bin
+              mkdir -p ${{targets.contextdir}}/etc/supervisor/conf.d
+              mkdir -p ${{targets.contextdir}}/usr/share/images/fluxbox
+          - runs: |
+              cp start-selenium-node.sh ${{targets.contextdir}}/opt/bin/
+              cp start-xvfb.sh ${{targets.contextdir}}/opt/bin/
+              cp selenium.conf ${{targets.contextdir}}/etc/supervisor/conf.d/
+              cp start-vnc.sh ${{targets.contextdir}}/opt/bin/
+              cp start-novnc.sh ${{targets.contextdir}}/opt/bin/
+              cp selenium_grid_logo.png ${{targets.contextdir}}/usr/share/images/fluxbox/ubuntu-light.png
+              cp generate_relay_config ${{targets.contextdir}}/opt/bin/
+
+              mkdir -p ${{targets.contextdir}}/home/$SEL_USER/.fluxbox
+              mkdir -p ${{targets.contextdir}}/tmp/.X11-unix
+              mkdir -p ${{targets.contextdir}}/home/$SEL_USER/.vnc
+              mkdir -p ${{targets.contextdir}}/opt/selenium
+              mkdir -p ${{targets.contextdir}}/opt/selenium
+              echo "${SEL_PASSWD}" | x11vnc -storepasswd - ${{targets.contextdir}}/home/$SEL_USER/.vnc/passwd
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+            - ${{package.name}}-config
+            - selenium-server
+            - geckodriver
+        environment:
+          SE_EVENT_BUS_PUBLISH_PORT: 4442
+          SE_EVENT_BUS_SUBSCRIBE_PORT: 4443
+          SE_EVENT_BUS_HOST: selenium-hub
+          CONFIG_FILE: /opt/selenium/config.toml
+          SE_BIND_HOST: true
+      pipeline:
+        - name: "Check selenium node-base status status"
+          uses: test/daemon-check-output
+          with:
+            start: "/opt/bin/start-selenium-node.sh"
+            timeout: 60
+            expected_output: |
+              Started Selenium node
+
+  # https://github.com/SeleniumHQ/docker-selenium/blob/trunk/NodeChrome/Dockerfile
+  - name: ${{package.name}}-node-chrome
```

Comment:
This is a duplicate of `-standalone-chromium` below. We should remove this and use that

Reply:
chrome package removed